src/api/dependenciesConfig.js

```diff
@@ -133,6 +147,20 @@ export function ensureIdentityExists(
       if (dep.autoActivateOnIdentityCreation) {
         ledger.activate(id);
       }
+      const weight = dep.autoInjectStartingPeriodWeight;
+      if (weight != null && !dep.periods.length) {
+        // Set default start period to one week from adding the dependency
```

Comment:
comment out of date. (i prefer this implementation which does as-of Date.now()).